great_expectations/expectations/metrics/column_aggregate_metrics/column_median.py

```diff
@@ -27,6 +27,11 @@ class ColumnMedian(ColumnAggregateMetricProvider):
     @column_aggregate_value(engine=PandasExecutionEngine)
     def _pandas(cls, column, **kwargs):
         """Pandas Median Implementation"""
+        metrics: Dict[str, Any] = kwargs.get("metrics") or {}
+        nonnull_count = metrics.get("column_values.nonnull.count")
+        if not nonnull_count:
+            return 0.0
```

Comment:
Nitpick - more personal preference but when not actually using the value, I think `if x not in metrics` is more appropriate. Happy to leave it as it though.